cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,69 @@
-describe('', () => {
-  before(() => {
+/// <reference types="cypress" />
 
-  });
+const { faker } = require('@faker-js/faker');
 
-  it('', () => {
+describe('article', () => {
+  beforeEach(() => {
+    cy.visit('user/register');
+  });
 
+  it('should create an article', () => {
+    const randomNumber = Math.random().toString().slice(2, 10);
+    const user = {
+      username: faker.lorem.word() + randomNumber,
+      email: faker.internet.email(),
+      password: faker.internet.password()
+    };
+    const article = {
+      title: faker.lorem.word(),
+      description: faker.lorem.words(3),
+      body: faker.lorem.words(10)
+    };
+    cy.login(user.email, user.username, user.password);
+    cy.createArticle(article.title, article.description, article.body)
+      .as('article');
+    cy.get('@article').should((resp) => {
+      expect(resp.status).to.eq(200);
+    });
```

Comment:
You need to check if the new article is visible on the profile page. Checking a response status is not enough.